src/mapnode.cpp

```diff
@@ -395,6 +395,77 @@ static std::vector<aabb3f> transformNodeBox(const MapNode &n,
 			boxes.push_back(box);
 		}
 	}
+	else if (nodebox.type == NODEBOX_CONNECTED)
+	{
+
```

Comment:
Ok, just to make sure why (I've never used reserve() before) - you want to do this to avoid many allocations and do only one, right?